### PR TITLE
Add proper sequence of setup.py dependencies

### PR DESCRIPTION
**Pull request by potiuk**
Because of our unstable Travis we merged a failing build
with #8512 and #8515 :(

---
Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

**Comment by potiuk** [comment]
Yeah. We really have to have the CI stable - this one was really stupidly missed.. 3 PRs left to merge :).

**Review by ashb — APPROVED** [review]

**Comment by mik-laj** [comment]
@potiuk  Github  Action failed. 😿

File: setup.py [hunk 1]
```diff
@@ -552,6 +552,8 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
 # DEPENDENCIES_EPOCH_NUMBER in the Dockerfile.ci
 #####################################################################################################
 INSTALL_REQUIREMENTS = [
+    'WTforms<2.3.0',
```
Comment (kaxil) [inline setup.py]: Why not change this to `wtforms<2.3.0` . `pip` is case-insensitive

Comment (kaxil) [reply setup.py]: https://www.python.org/dev/peps/pep-0426/#name

File: setup.py [hunk 2]
```diff
@@ -552,6 +552,8 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
...
+    # TODO: Remove after https://github.com/dpgaspar/Flask-AppBuilder/issues/1356 is fixed and released.
```
Comment (kaxil) [inline setup.py]:
```suggestion
```

File: setup.py [hunk 3]
```diff
@@ -598,8 +600,6 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
     'tzlocal>=1.4,<2.0.0',
     'unicodecsv>=0.14.1',
     'werkzeug<1.0.0',
-    'WTforms<2.3.0',
-    # TODO: Remove after https://github.com/dpgaspar/Flask-AppBuilder/issues/1356 is fixed and released.
 ]
```
Comment (kaxil) [inline setup.py:603]:
```suggestion
    'wtforms<2.3.0',	
    # TODO: Remove after https://github.com/dpgaspar/Flask-AppBuilder/issues/1356 is fixed and released.
]
```

File: setup.py (hunk 1 above)
Comment (potiuk) [reply setup.py]: Ah did not know that :). You learn every day ! Will change

Comment (potiuk) [reply setup.py]: Scary:

All comparisons of distribution names MUST be case insensitive, and MUST consider **hyphens and underscores** to be equivalent. :scream:

**Comment by potiuk** [comment]
> @potiuk Github Action failed.

An interesting thing why it failed. Seems boto team had some wrong release of boto/botocore packages boto3==1.12.43  yesterday and boto3==1.12.44 few minutes ago. Seems there was a problem with it and they released a new release.

**Comment by potiuk** [comment]
Superseded by #8522

**Pull request closed without merging**